### ISSUE TITLE
There is a problem with the lock key on the keyboard

### ISSUE THREAD

**Issue by wanglrebe** [sebanc/brunch#815]
Hello there.  I found that the lock key on the keyboard only works when a specific application is enabled (such as Text), and does not work when using other applications or on the desktop.

**Comment by wanglrebe** [comment]
lock key is F9
